### ISSUE TITLE
The Climb Trigger doesnt work

### ISSUE THREAD

**Issue by whit0r** [LibreVR/Revive#340]
The Climb - the trigger button in the emulated Touch Controllers doesnt seem to have any function, it shows pressed in the model in the hmd but you cant grab ledges with it...

**Comment by LibreVR** [comment]
Bullet train is fixed as a hotfix, please re-download the 1.0.1 release.

**Comment by LibreVR** [comment]
This issue is possibly related to #337

**Comment by LorkHang** [comment]
Latest version fixed this? Just had a session in The Climb and the Vive trigger works

**Comment by lghtanddrk** [comment]
Just tried it.  The triggers work but the grips don't. They show in the tutorial that they are being pressed but they don't work and you need them to chalk your hands.  The Y button doesn't function either but that might not be used for this game.  A is for jump.  Since I can't climb far I haven't confirmed that it actually works.

**Comment by LibreVR** [comment]
Please test the new release, it should fix the grip.

**Comment by lghtanddrk** [comment]
Thanks.  If I enabled the Revive auto update after installing ver. 1.02 how often does it look for new updates and will it always check before launching Steam VR?

**Comment by LibreVR** [comment]
It checks once per day automatically.

**Comment by LibreVR** [comment]
Closing this issue, since I'm pretty sure it is fixed. If you still encounter this issue, leave a comment and I'll re-open it.

**Comment by lghtanddrk** [comment]
You need to reopen this issue.  I just tried the latest version as you suggested and the grips still do not work to let you chalk your hands.

**Comment by LibreVR** [comment]
Thanks for the feedback, I was hoping this was the same issue as Toybox. I will look into it tonight.

**Comment by LorkHang** [comment]
they added a new mechanic that you need to shake the controller (for a few seconds) while holding the grip button, it works fine.

**Comment by lghtanddrk** [comment]
Ah, that explains it.  Thanks.  Any other changes?  Do you need to physically jump now while pressing the jump button?

**Comment by lghtanddrk** [comment]
Lorkhang, do you know how the jump mechanic works now that they added touch controls?  I can't seem to figure that one out either.

I also have seen some issues where the trigger suddenly stops working and you have to restart the game.

**Comment by LorkHang** [comment]
Jumping is the same as before which is the A button (Bottom side of right hand touchpad) ...ya I know...would be much more epic if they implement something like Climbey.

My last play through is about 30 minutes and I did not have any trigger issue.

**Comment by stephenrubio94** [comment]
I'm having a similar issue to @lghtanddrk .  In the tutorial segment, it says, "Press the grip button to grab a handheld", and I've tried the grip button, as well as every other button on my controllers, and nothing seems to work.  My in-game hands don't even seem to visibly react.  I tried this with and without ToggleGrip enabled through OpenVR advanced settings.  The tutorial does not say anything ab out chalking, this is literally step one.
